spark/v3.3/build.gradle

```diff
@@ -65,8 +65,9 @@ project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}") {
       exclude group: 'org.apache.avro', module: 'avro'
       exclude group: 'org.apache.arrow'
       exclude group: 'org.apache.parquet'
-      // to make sure io.netty.buffer only comes from project(':iceberg-arrow')
+      // to make sure netty libs only come from project(':iceberg-arrow')
       exclude group: 'io.netty', module: 'netty-buffer'
+      exclude group: 'io.netty', module: 'netty-common'
```

Comment:
We exclude `netty-common` from many deps but not all. I feel it is safer to be consistent.

Reply:
How about enforce the version instead of excluding here and there?

Reply:
I always consider forcing as the last resort. If we force a version, can it lead to unexpected consequences in libs that would normally pull in another version?

I'd be open to consider forcing but in a separate PR as this one is fixing the relocation in the current approach.

Reply:
I don't see any problem of enforcing in this context. Either works for me, though.